src/test/java/org/gridsuite/modification/server/modifications/tabularmodifications/TabularGeneratorModificationsTest.java

```diff
@@ -324,6 +324,237 @@ public void testSqlRequestsCountOnPostNetworkModificationsDuplicate2() throws Ex
         assertTabularModificationsEquals(modifications, idsMapping);
     }
 
+    /*
```

Comment:
Maybe we should move those tests to ModificationRepositoryTest or a new file GeneratorModificationRepositoryTest and make them work without any API calls. Can be done in another PR

Reply:
I think it's nice to test that through API calls no ? It's more like integration tests, but at least we're sure we test all DB operations end-to-end while calling the endpoint.